Showcase_Apps/VideoPlayerApp/metadata/Extensions/README.md

```diff
@@ -0,0 +1,45 @@
+# How to enable IntelliSense and Code Navigation in VS Code 
```

Comment:
We have covered this info in our public documentation. https://help.sap.com/doc/f53c64b93e5140918d676b927a3cd65b/Cloud/en-US/docs-en/guides/getting-started/mdk/vscode/intellisense-and-code-nav-support-for-extensions-metadata.html
You may want to just refer this link from your main read me file and remove this one.